src/main/java/de/adesso/kicker/match/controller/MatchController.java

```diff
@@ -67,11 +67,14 @@ public ModelAndView postAddMatch(@Valid Match match, BindingResult bindingResult
     }
 
     private ModelAndView defaultAddMatchView(ModelAndView modelAndView) {
-        modelAndView.addObject("match", new Match());
-        modelAndView.addObject("users", userService.getAllUsers());
-        modelAndView.addObject("currentUser", userService.getLoggedInUser());
-        modelAndView.addObject("notifications",
-                notificationService.getNotificationsByReceiver(userService.getLoggedInUser()));
+        var match = new Match();
```

Comment:
In my opinion, I would call the method on 'new Match()' and 'userService.getAllUsers()' directly as it looks a lot more cluttered that way and also it 'hides' the reason why you're using variables for the other one (which is fine).